materialize-sql/endpoint.go

```diff
@@ -92,6 +108,9 @@ type Endpoint struct {
 	NewTransactor func(ctx context.Context, _ *Endpoint, _ Fence, bindings []Table, open pm.Request_Open) (pm.Transactor, error)
 	// Tenant owning this task, as determined from the task name.
 	Tenant string
+	// ConcurrentApply of Apply actions, for system that may benefit from a scatter/gather strategy
+	// for changing many tables in a single publication.
```

Comment:
nit: tables are no longer changed in a publication.

Reply:
:+1: Kind of makes me wonder if supporting this concurrent pattern is even worth it anymore. But I guess its still nice for them to go fast if possible.